python/tchannel/event.py

```diff
@@ -40,11 +40,9 @@
     after_receive_request=0x21,
     before_receive_response=0x30,
     after_receive_response=0x31,
-    after_receive_system_error=0x40,
-    after_receive_system_error_per_attempt=0x41,
-    after_send_system_error=0x42,
-    on_operational_error=0x50,
-    on_operational_error_per_attempt=0x51,
```

Comment:
Ping. We should attempt to update the changelog as changes are made instead of trying to figure out what changes were made when releasing.